estate/models/estate_property.py

```diff
@@ -0,0 +1,52 @@
+from dateutil.relativedelta import relativedelta
+from odoo import fields, models
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate"
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(default=fields.Date.today()+relativedelta(months=3), copy=False)
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float(readonly=True, copy=False)
+    bedrooms = fields.Integer(default=2)
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
+    garden_orientation = fields.Selection(
+        selection=[('north', 'North'), ('south', 'South'), ('east', 'East'), ('west', 'West')]
+    )
+    active = fields.Boolean(default=True)
+    state = fields.Selection(
+        selection=[
+            ('new', 'New'),
+            ('offer_received', 'Offer Received'),
+            ('offer_accepted', 'Offer Accepted'),
+            ('sold', 'Sold'), ('cancelled', 'Cancelled'),
+        ],
+        copy=False,
+        default='new',
+        required=True,
+    )
+    property_type_id = fields.Many2one(
+        'estate.property.type', string='Property Type',
+    )
+    partner_id = fields.Many2one(
+        'res.partner', string='Buyer',
+    )
```

Comment:
Inline all of those. Let's stay consistent with the indentation we did previously.